projects/igniteui-angular/src/lib/grids/state.pivotgrid.spec.ts

```diff
@@ -168,7 +168,7 @@ describe('IgxPivotGridState #pivotGrid :', () => {
                             {
                                 "filteringOperands":[
                                     {
-                                        "condition": {"name":"in","isUnary":false,"iconName":"is-in","hidden":true},
+                                        "condition": {"name":"in","isUnary":false,"isNestedQuery": true,"iconName":"is-in","hidden":true},
```

Comment:
I am a little concerned that this is-in is not the same in condition we are refering to. We should check this in the base PR as well.